engine/sound/src/sound_decoder.h

```diff
@@ -59,7 +59,7 @@ namespace dmSoundCodec
          * Fetch a chunk of PCM data from the decoder. The buffer will be filled as long as there is
          * enough data left in the compressed stream.
```

Comment:
Perhaps update the comment to reflect the change, as "buffer" (singluar) is a bit misleading now since it's an array.